src/xo-app/vm/tab-network.js

```diff
@@ -135,173 +153,194 @@ class VifItem extends BaseComponent {
     }
   )
   _getIsNetworkAllowed = createSelector(
-    () => this.props.networkId,
+    () => createSelector(() => this.props.vif, vif => vif.$network),
```

Comment:
You can't do this. This needs to be a selector. And the selector here is simply:
```js
() => this.props.item.$network
```